tasks/rh-sign-image/rh-sign-image.yaml

```diff
@@ -85,8 +92,15 @@ spec:
                   -p manifest_digest=${manifest_digest} \
                   -p requester=$(params.requester) \
                   -p config_map_name=${config_map_name} \
-                  -t $(params.requestTimeout)
-            done
+                  -i $IDENTIFIER \
+                  -s false
+              ((++count))
 
-            echo "done"
+              if [ "$count" -eq "$N" ]; then
```

Comment:
I wonder how it would behave in case for instance: You have 5 requests, so 4 would be running in parallel. The 5th would start only when all 4 finishes, right? What if 3 finishes and one gets stuck? It would wait until all requests are done to start the 5th, correct?

Reply:
> You have 5 requests, so 4 would be running in parallel. The 5th would start only when all 4 finishes, right?

Correct.

> What if 3 finishes and one gets stuck? It would wait until all requests are done to start the 5th, correct?

If it gets stuck, eventually the `wait-for-ir` script will time out and the pipeline fails. The fifth ir will never get created.

> It would wait until all requests are done to start the 5th, correct?

Yes, it won't proceed to the next irs until the previous batch is done.

Reply:
I see. I think we should move forward this way but IMHO this should be more like a queue format - so we would always have "slots" available to create the next request when  another one is finished. I will write a proposal of this in the course of this week (using spare time) if you and @davidmogar  don't mind.